src/Freya.Core/Builder.fs

```diff
@@ -26,30 +26,30 @@ open Freya.Core.Operators
 
 (* Builder *)
 
-/// Defines a computation expression builder for constructing <see cref="Freya{T}" /> computations.
-type FreyaBuilder () =
+/// Defines a computation expression builder for constructing <see cref="Core{T}" /> computations.
+type CoreBuilder () =
 
-    member __.Return (t) : Freya<'T> =
-        Freya.returnM t
+    member __.Return (t) : Core<'T> =
+        Core.returnM t
     
-    member __.ReturnFrom (m: Freya<'T>) =
+    member __.ReturnFrom (m: Core<'T>) =
         m
     
-    member __.Bind (m1: Freya<'T>, m2: 'T -> Freya<'U>) : Freya<'U> =
-        Freya.bind m1 m2
+    member __.Bind (m1: Core<'T>, m2: 'T -> Core<'U>) : Core<'U> =
+        Core.bindM m1 m2
```

Comment:
I'd rather remove the `M`. Perhaps rename `returnM` to `wrap` or `result` or `init`, the latter of which is probably the more common case. See `Array.init`.